.github/workflows/deploy.yml

```diff
@@ -39,13 +45,25 @@ jobs:
 
     - name: Publish package to PyPI
       uses: pypa/gh-action-pypi-publish@release/v1
-      with:
-        password: ${{ secrets.pypi_token }}
 
     - name: Set up Python
```

Comment:
This needs to be in the `release-notes` job no?

Reply:
fixed, thanks

every time i try the web ux for something trivial i demonstrate its a bad idea for me to do that :(

going to use local